index.html

```diff
@@ -64,15 +63,15 @@ <h1>Bikes for Refugees</h1>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="donate-button">Donate a bike today</button>
+            <button class="volunteer-button">Volunteer</button>
           </div>
 
           <div>
             <h2 class="heading-underline">Learn more</h2>
```

Comment:
![Screenshot 2023-09-26 at 09 14 31](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/7f3b476c-6d5b-4b6e-9323-0c444d279094)

Can you create some space about the learn more section?

Reply:
yes i can